### PR TITLE
docs: supplement observability guide

### PR DESCRIPTION
**Pull request by chunshao90**
Supplement observability guide.

### CODE REVIEW

File: docs/en/src/operation/observability.md [hunk 1]
```diff
@@ -2,5 +2,97 @@
...
+    -p 9090:9090 \
+    -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus:v2.41.0
+```
+For more detailed installation methods, refer to [here](https://prometheus.io/docs/prometheus/latest/installation/).
+
+
+## Grafana
+
+[Grafana](https://github.com/grafana/grafana) is a open and composable observability and data visualization platform.
```
Comment (MachaelLee) [inline docs/en/src/operation/observability.md:45]: `a` to `an`

File: docs/src/operation/observability.md [hunk 2]
```diff
@@ -2,5 +2,97 @@
...
+Two ceresdb http service are started on localhost:5440 and localhost:5441.
+
+```yaml
+global:
+  scrape_interval: 30s
+
+scrape_configs:
+  - job_name: ceresdb-server
+    static_configs:
+    - targets: [your_ip:5440, your_ip:5441]
```
Comment (jiacai2050) [inline docs/src/operation/observability.md:22]:
```
localhost:5440
```

Comment (chunshao90) [reply docs/src/operation/observability.md]: In the container, set to localhost, prometheus cannot access services outside the container.

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,9 @@
+install:
+	cargo install mdbook
+	cargo install mdbook-i18n --git https://github.com/chunshao90/mdbook-i18n.git --rev 23c364c6a207d7beb8f8d2bf0b563ca55c367b44
```
Comment (jiacai2050) [inline Makefile:3]: Update install cmd in CI.

**Review by MachaelLee — APPROVED** [review]
LGTM

**Pull request merged**